app/src/main/admin/actions/UserActions.ts

```diff
@@ -2,6 +2,28 @@ import { alt } from "../../shared/alt";
 import { FetchActions, FetchActionsInterface } from "../../shared/actions/FetchActions";
 import { User } from "../../shared/models/Generated";
 
-export const userActions = alt.createActions<FetchActionsInterface<User[]>>(
-    class extends FetchActions<User[]> {}
-);
+interface Actions extends FetchActionsInterface<User[]> {
+    setCurrentUser(username: string): string;
+    beginFetchDetails(username: string): string;
+    updateUserDetails(details: User): User;
+
+}
```

Comment:
I've been trying to move away from using `FetchActionsInterface` if there is more than one set of `beginFetch` and `update` on the class.

So rather than having the slightly confusing: `beginFetch`, `update`, `beginFetchDetails`, `updateDetails`, I'd prefer to have `beginFetchUsers`, `updateUsers`, `beginFetchDetails`, `updateDetails`.